rpcs3/rpcs3qt/pkg_install_dialog.h

```diff
@@ -0,0 +1,13 @@
+﻿#pragma once
+
+#include <QDialog>
+
+#include <functional>
+
+class pkg_install_dialog : public QDialog
+{
+	Q_OBJECT
+
+public:
+	explicit pkg_install_dialog(std::function<bool(const QString&, bool)> installCb, const QStringList& paths, QWidget* parent = nullptr);
```

Comment:
The callback should be the last param.
Also snake_case

Reply:
On second thought. The callback is bad style. Just use a signal slot mechanic

Reply:
I thought about this, but what best connects to what here? I think I can't cleanly access `main_window` from `pkg_install_dialog`, so the best idea I have is to connect `main_window::InstallPKG` to `pkg_install_dialog::PKGInstallRequested` after constructing `pkg_install_dialog` in `main_window::dropEvent`, but before `dlg.exec()`. Do you think it's good design?

Reply:
Maybe Just pass a string list to the dialog and install these after accept signal

Reply:
You mean by connecting `main_window::InstallPkg` to `pkg_install_dialog::accepted`? Sounds good.

Reply:
Came up with something, looks good to me.